setup.py

```diff
@@ -40,7 +40,7 @@
 # Give setuptools a hint to complain if it's too old a version
 # 30.3.0 allows us to put most metadata in setup.cfg
 # Should match pyproject.toml
-SETUP_REQUIRES = ["setuptools >= 30.3.0"]
+SETUP_REQUIRES = ["setuptools >= 38.3.0"]
```

Comment:
For my info -- should be ok even if we start providing packages in neurodebian for debian stable buster which as 40.8.0-1:
```
lena:~
$> whohas -D Debian,Ubuntu --strict python3-setuptools
Ubuntu      python3-setuptools                     20.7.0-1           86K             all                       http://packages.ubuntu.com/xenial/python3-setuptools
Ubuntu      python3-setuptools                     39.0.1-2           242K            all                       http://packages.ubuntu.com/bionic/python3-setuptools
Ubuntu      python3-setuptools                     41.1.0-1           247K            all                       http://packages.ubuntu.com/eoan/python3-setuptools
Ubuntu      python3-setuptools                     45.2.0-1           322K            all                       http://packages.ubuntu.com/focal/python3-setuptools
Ubuntu      python3-setuptools                     46.1.3-1           318K            all                       http://packages.ubuntu.com/groovy/python3-setuptools
Debian      python3-setuptools                     5.5.1-1            131K            all                       http://packages.debian.org/jessie/python3-setuptools
Debian      python3-setuptools                     33.1.1-1           210K            all                       http://packages.debian.org/stretch/python3-setuptools
Debian      python3-setuptools                     40.8.0-1           298K            all                       http://packages.debian.org/buster/python3-setuptools
Debian      python3-setuptools                     46.1.3-1           373K            all                       http://packages.debian.org/bullseye/python3-setuptools
Debian      python3-setuptools                     46.1.3-1           373K            all                       http://packages.debian.org/sid/python3-setuptools
```